aten/src/ATen/native/native_functions.yaml

```diff
@@ -339,6 +339,18 @@
     CPU: _bincount_cpu
     CUDA: _bincount_cuda
 
+- func: bitwise_not(Tensor self) -> Tensor
+  variants: function, method
+
+- func: bitwise_not_(Tensor(a!) self) -> Tensor(a!)
+  variants: function, method
```

Comment:
The in-place version should be method-only

Reply:
Hmm... There are several other in-place versions with `function` variant. Should they also be changed?

Reply:
For example, `abs_`, `ceil_`

Reply:
Yes, they should probably be changed (but not as part of this PR)

cc @gchanan

Reply:
Sure -- Just to confirm I need to make these changes. Updated.